### DISCUSSION TITLE
How to know what pool/srp device is bound to?

### DISCUSSION THREAD

**Discussion by 212850a** [dell/PyU4V#167] [General]
Hello, 
After spending some time I could not find the way to identify what pool/srp each device is bound to, get_volume() function does not provide these details.
May be I miss something or there is no way to do it via RestAPI?

From solutions enabler it's available from "symdev show" command output like:
```
symdev -sid 1111 show DEVICE
...
Bound Pool Name:   SRP_NAME
...
```

**Comment by rawstorage** [comment]
Hi, there is usually only a single SRP on the majority of PowerMax arrays, usually called SRP_1,  this is reported at the storage group level.  If you do a get_storage_group_details on the storage group you will find the bound SRP
{
  "storageGroupId": "ansible_metro",
  "slo": "Diamond",
  "service_level": "Diamond",
  "base_slo_name": "Diamond",
  "srp": "SRP_1",
  "slo_compliance": "STABLE",
  "num_of_vols": 2,
  "num_of_child_sgs": 0,
  "num_of_parent_sgs": 0,
  "num_of_masking_views": 0,
  "num_of_snapshots": 0,
  "cap_gb": 4,
  "device_emulation": "FBA",
  "type": "Standalone",
  "unprotected": true,
  "compression": true,
  "compression_ratio_to_one": 0,
  "vp_saved_percent": 100,
  "unreducible_data_gb": 0
}

**Comment by 212850a** [comment]
Thanks for confirmation, Paul. It makes sense.
